src/Modules/Workspaces/Commands.Workspaces/SetPowerBIWorkspace.cs

```diff
@@ -30,7 +30,7 @@ public class SetPowerBIWorkspace : PowerBIClientCmdlet, IUserScope
 
         [Parameter(Mandatory = false, ParameterSetName = PropertiesParameterSetName)]
         [Parameter(Mandatory = false, ParameterSetName = WorkspaceParameterSetName)]
-        public PowerBIUserScope Scope { get; set; } = PowerBIUserScope.Individual;
+        public PowerBIUserScope Scope { get; set; } = PowerBIUserScope.Organization;
```

Comment:
Out of curiosity, when this API exists for the user and we update the cmdlet accordingly, will we switch the default back to Individual? I understand the reasoning behind this change, but it could be confusing for the end user.

Reply:
I see leaving Set-PowerBIWorkspace as default Individual but Restore cmdlet will always be fore Organization. I will witch it back and file a bug we should also support user updates as well

Reply:
If this is the case for the Restore cmdlet, can we remove the scope parameter altogether?

Reply:
It would be a breaking change if someone is calling it already but the likelihood of anyone using it as preview workspaces are starting to roll out is small

Reply:
I switched the default back for this